content/_policy/contact._en.md

```diff
@@ -25,7 +25,7 @@ help_center_content: >-
   * [I need to change my information or manage my account](/help/manage-your-account/overview/)
 
   * [Browse more help articles](/help/)
-
+maintenance_window_content: Login.gov’s Contact Center is currently undergoing maintenance from <strong>January 20th, 2023 9 P.M. EDT - January 21st, 2023 1 A.M. EDT.</strong> Please visit our FAQs below for assistance.
```

Comment:
Could we re-use any of what we built for the handbook? https://github.com/18F/identity-handbook/pull/341

Reply:
As in, could we rely on the IntlDate API to do some approximately good formatting of times?

Reply:
Yeah, I think we could do something like that. The formatting is _close_, but not identical to the approved content. Given the timeframe for this request, I'm inclined to keep it hard-coded for now, and revisit once we've had a chance to get some buy-in from UX. It's also not totally clear to me if we expect us to need to keep this code around long-term anyways (i.e. are we expecting continued future maintenance?).

```ts
const date = new Date('2023-01-21T06:00:00Z')
const options = { year: 'numeric', month: 'long', day: 'numeric', hour: 'numeric', minute: 'numeric', timeZoneName: 'short' };

new Intl.DateTimeFormat('en-US', options).format(date);
// January 21, 2023 at 1:00 AM EST

new Intl.DateTimeFormat('es-MX', options).format(date);
// 21 de enero de 2023, 01:00 GMT-5

new Intl.DateTimeFormat('fr-CA', options).format(date);
// 21 janvier 2023 à 01 h 00 HNE
```

Reply:
Revisiting this since I chatted with @2hsieh for some other content revisions and brought this up. I pushed changes in e897534 to dynamically add the formatted date to the alert.